eppo/src/androidTest/java/cloud/eppo/android/EppoClientTest.java

```diff
@@ -1037,7 +1037,7 @@ public void testAssignmentEventCorrectlyCreated() {
     Date parsedTimestamp = capturedAssignment.getTimestamp();
     assertNotNull(parsedTimestamp);
     assertTrue(parsedTimestamp.after(testStart));
-    assertTrue(parsedTimestamp.before(assertionDate));
+    assertTrue(parsedTimestamp.before(assertionDate) || parsedTimestamp.equals(assertionDate));
```

Comment:
If the test is executed really fast, the parsed and assertion timestamps will be equal, so changing this to an inclusive `<=` check so the test is not flakey.